### PR TITLE
website/integrations: proxmox-ve: update flow selection

### PR DESCRIPTION
**Pull request by Webstijlen**
The option needs to be selected, or else auth fails

## Details

REPLACE ME

---

## Checklist

-   [ ] Local tests pass (`ak test authentik/`)
-   [ ] The code has been formatted (`make lint-fix`)

If an API change has been made

-   [ ] The API schema has been updated (`make gen-build`)

If changes to the frontend have been made

-   [ ] The code has been formatted (`make web`)

If applicable

-   [ ] The documentation has been updated
-   [ ] The documentation has been formatted (`make website`)

### CODE REVIEW

File: website/integrations/services/proxmox-ve/index.md [hunk 1]
```diff
@@ -26,6 +26,7 @@ The following placeholders will be used:
 Under _Providers_, create an OAuth2/OpenID provider with these settings:
 
 -   Name: proxmox
+-   Authentication Flow: default-authentication-flow (Welcome to authentik!)
```
Comment (tanberry) [inline website/integrations/services/proxmox-ve/index.md:29]:
```suggestion
-   Authentication Flow: `default-authentication-flow` (Welcome to authentik!)
```

**Review by tanberry — CHANGES_REQUESTED** [review]
Thanks @Webstijlen for that important clarification. Only one small nit (format the name of flow), please.

Can you also add a Head 2 (##) Right above Step 1, titled "authentik configuration"? I see now that Portainer and Rancher (other integrations listed beside Proxmox) do not follow our template either. Take a look at the one for ArgoCD... that's what we need, two separate areas for authentik setup and for the integrating product setup.

Let us know of questions, thanks again!

**Review by BeryJu — CHANGES_REQUESTED** [review]
This option should not affect whether authentication succeeds or fails, and if it does that's a bug. Could you provide the logs/error messages that happen when this option is not selected and you try to authenticate?

**Comment by BeryJu** [comment]
Having now setup proxmox myself and having set it up with OIDC, this change is not required

**Pull request closed without merging**